src/overlays/actors/ovl_En_Dinofos/z_en_dinofos.c

```diff
@@ -243,10 +243,10 @@ static DamageTable sDamageTable = {
 static CollisionCheckInfoInit sColChkInfoInit = { 4, 40, 100, 80 };
 
 static TexturePtr D_8089E33C[] = {
-    object_dinofos_Tex_008E30,
-    object_dinofos_Tex_009030,
-    object_dinofos_Tex_009230,
-    object_dinofos_Tex_009030,
+    gDinolfosEyeOpenTex,
+    gDinolfosEyeHalfTex,
+    gDinolfosEyeClosedTex,
+    gDinolfosEyeHalfTex,
```

Comment:
Since you know all of these are eye textures you could probably rename D_809E33C

Reply:
oh true hadnt considered that